### PR TITLE
Pull Android Docker in GCB

### PR DESCRIPTION
**Pull request by ipetr0v**
This change:
- Adds `docker_pull` for Android SDK in the GCB
- Updates the text field in the hello-world Android application

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -7,12 +7,24 @@ steps:
     waitFor: ['-']
     timeout: 10m
     args: ['pull', 'gcr.io/oak-ci/oak:latest']
+  # Pull Android Docker image.
```
Comment (tiziano88) [inline cloudbuild.yaml]: I would keep the Android stuff at the end, in fact perhaps adds an extra line between the current steps and the android steps. I'm thinking we may want to split the triggers into two separate cloudbuild files in the short future. WDYT?

File: examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java [hunk 2]
```diff
@@ -46,8 +46,8 @@ public void onCreate(Bundle savedInstanceState) {
 
     // Set default address.
     // Android emulator forwards `10.0.2.2` to the host machine.
```
Comment (tiziano88) [inline examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java]: Where does this `10.0.2.2` come from? Could you link to some documentation?

File: cloudbuild.yaml (hunk 1 above)
Comment (ipetr0v) [reply cloudbuild.yaml]: Done

File: examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java (hunk 2 above)
Comment (ipetr0v) [reply examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java]: Done

**Pull request merged**